### ISSUE TITLE
Accessibility: MAS40B: Inspect: Inappropriate 'Name' property for "Path Text Box" in 'Select Destination Location' window in Visual Studio Code editor installation wizard.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/vscode#17173]
VS Code Version:1.6.1
OS Version:1607

**Steps to Reproduce:**
1. Open Visual Studio Code editor installation wizard.
2. Click Next and Go to 'Select Destination Location' window.
3. Tab to the "Path" text box besides "Browser" button.
4. Using inspect, check it's 'Name' property.

Note:
Issue also exists for Text box in next window 'Select Start Menu Folder' of the installation.

**Expected Result:**
' 'Name' property should be appropriate and meaningful.

**Actual Result:** 
'Name' property is not meaningful. Inspect shows "To continue, click Next. If you would like to select a different folder, click Browse" as Name property.
Narrator Info:-  Narrator announces like "Tab
To continue, click Next. If you would like to select a different folder, click Browse., editing text, C:\Program Files (x86)\Microsoft VS Code".
NVDA Info:-  NVDA announces like "To continue, click Next. If you would like to select a different folder, click Browse., text, Selected C:\Program Files (x86)\Microsoft VS Code"

**User Impact:**
Users relying on screen readers will get confused if the name property is not meaningful and inappropriate.

![browse name prop 2](https://cloud.githubusercontent.com/assets/22956130/21184116/00cdb37a-c231-11e6-9187-39f6bd991975.png)

![browse name prop](https://cloud.githubusercontent.com/assets/22956130/21184124/03d8ea62-c231-11e6-938b-77a2267943a2.png)
![path no name](https://cloud.githubusercontent.com/assets/22956130/21184127/06700a8a-c231-11e6-9c34-e95f01e5050c.png)

**Comment by joaomoreno** [comment]
Won't do anything in the Setup for accessibility.

**Comment by chrisdias** [comment]
@joaomoreno can we file an upstream issue on Inno Setup?

**Comment by joaomoreno** [comment]
Let me just confirm something:

**Comment by joaomoreno** [comment]
Created an upstream issue: http://news.jrsoftware.org/read/article.php?id=105213&group=jrsoftware.innosetup#105213

**Comment by joaomoreno** [comment]
To keep the number of issues in our inbox on a manageable level, we're closing issues that have been on the backlog for a long time but haven't gained traction: We look at the number of votes the issue has received and the number of duplicates issues filed.

Thanks for your understanding. Happy coding!
